src/apps/viewer/app.ts

```diff
@@ -423,6 +423,28 @@ export class Viewer {
         });
     }
 
+    async loadFullResolutionMap(emdbId: string, iso: number) {
+        const plugin = this.plugin;
+        const numericId = parseInt(emdbId.toUpperCase().replace('EMD-', ''));
+        const url = `https://ftp.ebi.ac.uk/pub/databases/emdb/structures/EMD-${numericId}/map/emd_${numericId}.map.gz`;
+        const data = await plugin.build().toRoot()
+            .apply(StateTransforms.Data.Download, { url, isBinary: true, label: emdbId }, { state: { isGhost: true } })
+            .apply(StateTransforms.Data.DecompressGzip, { })
```

Comment:
```suggestion
            .apply(StateTransforms.Data.DecompressGzip)
```

Should work like this